### ISSUE TITLE
Cannot page through viewer (UV) with only the keyboard.

### ISSUE THREAD

**Issue by sdellis** [pulibrary/pulfalight#496]
Perhaps we should look into using Mirador as our IIIF viewer as the UV is not fully accessible. Alternative is to submit a PR to the UV codebase.

Severity: Major

WCAG Guideline:
2.1.1 A: All functionality is available from a keyboard, and does not require specific timings for keystrokes.

**Comment by hackartisan** [comment]
A lot of accessibility work has happened in UV recently. We haven't updated it in a while. UV lives in figgy.

**Comment by tpendragon** [comment]
Did we need this done before launch, so it got moved to the priority column? It seems unlikely we'll be able to get it done in time.

**Comment by sdellis** [comment]
My understanding is that we want to do as much as we can to meet the WCAG 2.1 guidelines for PulfaLight. @tpendragon I agree that getting changes to the UV in time (without forking) would be impossible before our deadline. What about the use of Mirador 3 (at least for non-AV materials), which already meets the accessibility guidelines? Is this a question for @kelynch?

**Comment by kelynch** [comment]
I think it makes sense to try to update the Universal Viewer and stay in line with its accessibility updates, and agree that it's something we should focus on after this week.

**Comment by ccleeton** [comment]
Has the viewer been updated since this ticket?

**Comment by tpendragon** [comment]
No.
